grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/GRpcServerRunner.java

```diff
@@ -45,6 +47,8 @@
 
     private Server server;
 
+    private CountDownLatch latch = new CountDownLatch(1);
```

Comment:
@jvmlet @dounan hi guys, don't you mind if I participate in this PR as a reviewer? :) I have a small note.
How do you think about to set this field as final + initialize it inside the constructor? Because you will use it in several threads.

Reply:
Thanks @mchernyakov for the note. I do agree that `latch` should be final, but why is it matter to initialize it in constructor ?

Reply:
@jvmlet it is not a necessity, but I've just thought that it could be nice to have a common style inside this class, for instance, on with field `private final ServerBuilder<?> serverBuilder;` you also have initialization in the constructor (obviously, because of a parameter). 

Up to you :)